src/components/ArticlesView/ArticlesView.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import api from "../../../api";
+import ArticleCard from "./ArticleCard";
+
+const ArticlesView = () => {
+    const [articles, setArticles] = useState([]);
+
+    api.get('/articles?sort_by=comment_count')
+    .then((response) => {
+        setArticles(response.data.articles);
+    })
+    .catch((err) => {
+        console.log(err);
+    });
```

Comment:
This api request should not be used like this. This should be inside of a useEffect so we can control how and when this request is re-triggered. We should also not be making the request here like this and should create a function in your api.js file that makes this request to this endpoint and returns the data we are wanting.

I would also suggest not sorting specifically by comment_count at this stage as it should be sorted by created_at by default. This is also not how queries should be made using axios. I would recommend looking at the following notes on how to do this correctly:

l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=axios-params